compiler-rt/lib/scudo/standalone/linux_common.cpp

```diff
@@ -0,0 +1,65 @@
+//===-- linux_common.cpp ----------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "platform.h"
+
+#if SCUDO_LINUX
+
+#include "common.h"
+#include "internal_defs.h"
+#include "linux_common.h"
+#include "string_utils.h"
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+namespace scudo {
+
+void NORETURN die() { abort(); }
```

Comment:
nit: If this is only used here, I may mark it as static and put it in the global namespace.

Reply:
It's used by the report code. I moved it back into the linux.cpp and trusty.cpp files since it feels like it better belongs there. Even though the die on trusty and linux are the same, it's possible they could be changed so leave it